habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -469,6 +469,163 @@ def step(self, *args, is_last_action, **kwargs):
             return {}
 
 
+@registry.register_task_action
+class BaseVelSpotAction(ArticulatedAgentAction):
```

Comment:
Should this class inherit from BaseVelAction? Then maybe some of the parameters below do not need to be re-defined?

Reply:
good point! I will think about this

Reply:
Done! thank you!